src/doc/gui_manual/Operators/OperatorTypes/PICS/Poincare_operator.rst

```diff
@@ -0,0 +1,318 @@
+"""""""""
+
+While integrating the streamline to be used the for Poincaré plot, the user has
+the option to require a minimum number of initial puncture points for the
+analysis. Similarly they may limit it in case of run-a-way fieldlines that can
+not be fully analyzed.
+
+Minimum
+    Minimum number of punctures for the analysis.
+
+Maximum
+    Maximum number of punctures for the analysis. 
+
+Puncture plot type
+    The type of the puncture lot. Options are:
+
+    * Single - the analysis is based on the standard double periodic system
+      (toroidal-poloidal periodicity)
+    * Double - the analysis is based on the double Poincaré plot. In addition
+      to the toroidal-poloidal periodicity a third periodicity exists that is
+      based on the integration time. 
+
+    When selecting double Poincaré plot puncture points are accepted if and
+    only if the period is within the tolerance of the period (the period is
+    set as part of the Poincaré Pathline Options). 
+
+       * Period tolerance - when an integral curve punctures the plane, the
+         period must be within the tolerance value. 
+
+Maximum Number of steps
+   Because of the possibility for very long integration times integration
+   limits must be utilized.
+
+Puncture Plane
+    The typical puncture plane is the poloidal plane but in some cases seeing
+    the punctures in the toroidal plane may be useful.
+
+Note: when selecting “Toroidal” the “Analysis” must also be set to “Punctures
+only” as there is currently no analysis in the toroidal plane. See below.
+
+
+Analysis
+^^^^^^^^
+
+Analysis Type
+"""""""""""""
+
+The user may turn the analysis on or off. Options are:
+
+None - Puncture only
+    This will result in constructing a traditional Poincaré plot using only
+    points. 
+
+Full
+    This will analyze each fieldline’s geometry and attempt to reconstruct the
+    cross sectional profile of the surface which the fieldline lies on.
+    Further, the analysis attempt to identify the topology of the surface. 
+
+Maximum toroidal winding
+    Limit the search of the toroidal winding to lower order values. Zero
+    indicates no limit.
+
+Override toroidal winding
+    In some cases, such as debugging, it may be informative to force the
+    toroidal winding to have a set value. Zero indicates no override. 
+
+Override poloidal winding
+    In some cases such as debugging, it may be informative to force the
+    poloidal winding to have set value. Zero indicates no override.
+
+Winding pair confidence (Range 0-1, Default 0.9)
+    Sets the limit for the number of mismatches in the consistency in the
+    winding pairs.
+
+Detect Rational Surface
+    Allows for the construction of rational surfaces via an iterative process.
+
+    * Maximums iterations – sets the number of iterations for constructing the
+    rational surface. Typically, it can be found with 5-10 iterations. 
+
+Note: The rational surface construction is experimental code and does not
```

Comment:
Make this a `.. danger:` note in Sphinx. Consider other cases in these docs where either `danger` or `warn` might be appropriate.